core/grid.ts

```diff
@@ -32,11 +32,11 @@ export class Grid {
    * This should always be the same as the workspace scale.
    */
   private scale_ = 1;
```

Comment:
why not this one?

Reply:
Oops. Because I didn't push the commit that removed that property entirely--it was set but never used.